content/javascript/concepts/bitwise-operators/bitwise-operators.md

```diff
@@ -37,6 +37,19 @@ console.log(19 & 7); // Output: 3
 // 00011 =  3
 ```
 
+## Codebyte Example
+
+The following example illustrates the use of Bitwise `&` operator:
```

Comment:
Let the author know they can try executing this code themselves, and update this line accordingly (For other codebytes too.).

Reply:
Thanks for your review. Changes have been effected! 😊